ActindoWorkFlow/Operation/03_CompareWorkflows.md

```diff
@@ -0,0 +1,178 @@
+# Compare the workflows versions
+
+The comparison function of the workflow enables to compare different workflow versions. By doing this, you can compare the JSON code lines of two different workflow versions, whereby all differences between the relevant code lines are highlighted.  
+
+You can use this function to do the following: 
+- Compare a specific workflow version to any other version of that workflow available on the current instance.
+- Compare a specific workflow version to any workflow version available on another instance. 
+- Change, insert, or delete code lines. This is possible, because you can edit the JSON code directly.  
+- Display change logs. By doing it, you are able to do software-audits.
+
+## Open the comparison window
+
+Open the comparison window to compare different workflow versions. There are two different ways to open it.
+
+#### Prerequisites
+
+You have the required rights to edit a workflow.
+
+#### Procedure
+
+*Process Orchestration > Workflows > Tab OVERVIEW*
+
+![Workflows](../../Assets/Screenshots/ActindoWorkFlow/Workflows/Workflows.png "[Workflows]")
+
+Select the workflow for which you want to do a comparison.   
+The *Workflow versions* view is displayed.
+    
+![Workflow versions](../../Assets/Screenshots/ActindoWorkFlow/Workflows/WorkflowVersions.png "[Workflow versions]")
+
+**Open the comparison window via the Workflow versions view**
+
+1. Click the checkbox at the desired version you want to use for comparison.   
```

Comment:
Ist vielleicht "the desired versions", also in plural? Man braucht zwei Versionen zum Vergleich. 

Beim Testen in der Sandbox: wenn ich die letzte Version (13) auswähle, wird mit der vorletzten (12) verglichen. Wenn ich aber eine beliebige Version aus der Mitte auswähle (z.B. 8), wird sie mit der letzten verglichen. Ist es immer so? Sollten wir das nicht erwähnen? Ich denke, das kann man nicht intuitiv herausfinden.